pyInterface/genpw.py

```diff
@@ -24,6 +24,8 @@
 	parser.add_argument("-M", type=float, metavar="#", dest="massLowerBinBoundary", help="lower boundary of mass range in MeV (overwrites values from reaction file)")
 	parser.add_argument("-B", type=float, metavar="#", dest="massBinWidth", help="width of mass bin in MeV")
 	parser.add_argument("-u", "--userString", type=str, metavar="#", dest="userString", help="metadata user string", default="")
+	parser.add_argument("--massAndtPrimeVariableNames", type=str, dest="massAndtPrimeVariableNames", help="Name of the mass and t' variable (default: 'mass,tPrime')", default="mass,tPrime")
+	parser.add_argument("--noStoreMasstPrime", action="store_true", dest="noStoreMasstPrime", help="Do not store mass and t' variable of each event.") 
```

Comment:
the names `massAndtPrimeVariableNames` and `noStoreMasstPrime` are rather long; the order of `tPrime` and `mass` differs from the C++ part of the program; once its with `and`, once without; if `tPrime` occurs in the middle of a variable or function name it should probably be written with a capital `TPrime`